src/Analyzer.Core/TemplateAnalyzer.cs

```diff
@@ -114,42 +155,96 @@ public IEnumerable<IEvaluation> AnalyzeTemplate(string template, string template
 
             var templateContext = new TemplateContext
             {
-                OriginalTemplate = JObject.Parse(template),
+                OriginalTemplate = JObject.Parse(rootTemplate),
                 ExpandedTemplate = templatejObject,
                 IsMainTemplate = true,
                 ResourceMappings = armTemplateProcessor.ResourceMappings,
                 TemplateIdentifier = templateFilePath,
                 IsBicep = isBicep,
-                SourceMap = sourceMap
+                SourceMap = sourceMap,
+                PathPrefix = pathPrefix
             };
 
             try
             {
                 IEnumerable<IEvaluation> evaluations = this.jsonRuleEngine.AnalyzeTemplate(templateContext);
-                evaluations = evaluations.Concat(this.powerShellRuleEngine.AnalyzeTemplate(templateContext));
 
-                // For each rule we don't want to report the same line more than once
-                // This is a temporal fix
-                var evalsToValidate = new List<IEvaluation>();
-                var evalsToNotValidate = new List<IEvaluation>();
-                foreach (var eval in evaluations)
+                if (this.powerShellRuleEngine != null)
```

Comment:
The merge with development wasn't done properly, see how we don't do this if anymore: https://github.com/Azure/template-analyzer/blob/9bb87364dab126ab67f28bc822fa7ccb8ef60db5/src/Analyzer.Core/TemplateAnalyzer.cs#L129